test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -20,10 +20,22 @@ Describe "Basic Auth over HTTP not allowed on Unix" -Tag @("CI") {
         $err = ({New-PSSession -ComputerName 'localhost' -Credential $credential -Authentication Basic}  | Should -Throw -PassThru  -ErrorId 'System.Management.Automation.Remoting.PSRemotingDataStructureException,Microsoft.PowerShell.Commands.NewPSSessionCommand')
         $err.Exception | Should -BeOfType [System.Management.Automation.Remoting.PSRemotingTransportException]
         # Should be PSRemotingErrorId.ConnectFailed
-        # Ensures we are looking at teh expected instance
+        # Ensures we are looking at the expected instance
         $err.Exception.ErrorCode | Should -Be 801
     }
 
+    It "New-PSSession should NOT throw a ConnectFailed exception when specifying Basic Auth over HTTPS on Unix" -skip:($IsWindows) {
+        $password = ConvertTo-SecureString -String "password" -AsPlainText -Force
```

Comment:
In the `HelpersRemoting.psm1` module, I believe you can get real creds to validate it works.

Reply:
I actually don't want to establish a connection; I'm verifying that Basic Auth verification doesn't reject the request.

Reply:
Is there a separate test that validates it works?

Reply:
@SteveL-MSFT: No, we don't have any automated remoting tests for PSRP at this point.